spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/ExpressionEvaluatingSqlParameterSourceFactory.java

```diff
@@ -158,7 +158,7 @@ public void afterPropertiesSet() throws Exception {
 
 		private final boolean cache;
 
-		private ExpressionEvaluatingSqlParameterSource(Object input, Map<String, ?> staticParameters,
+		ExpressionEvaluatingSqlParameterSource(Object input, Map<String, ?> staticParameters,
```

Comment:
OK. So, any `private` classes must not contain `private` ctor or be without ctor at all?